webdrivercss-pull64.patch

```diff
@@ -0,0 +1,5 @@
+21c21,22
+<             excludeRect: []
+---
+>             excludeRect: [],
+>             scrollPos: {x: 0, y:0}, 
```

Comment:
Yeah, they merged it like 10 hours ago. It's been sitting around for weeks, so... cool!